README.md

```diff
@@ -2433,22 +2695,28 @@ Plot.formatIsoDate(new Date("2020-01-01T00:00.000Z")) // "2020-01-01"
 
 Given a *date*, returns the shortest equivalent ISO 8601 UTC string. If the given *date* is not valid, returns `"Invalid Date"`.
 
-#### Plot.formatWeekday(*locale*, *format*)
+<!-- jsdoc formatWeekday -->
+
+#### Plot.formatWeekday(*locale*, *weekday*)
```

Comment:
change of argument name

Reply:
Same here, we use `format` internally in the function. Perhaps I can rename `const format` to something else so we can use `format` instead of `weekday`?

Reply:
yes, I'd favor that in this case